pkg/metrics/metrics.go

```diff
@@ -206,57 +212,74 @@ var (
 		Help:      "Number of errors that occurred in the datapath or datapath management",
 	},
 		[]string{LabelDatapathArea, LabelDatapathName, LabelDatapathFamily})
+
+	// Health statistics
+
+	// Latency is a histogram showing latency to nodes
+	Latency = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Namespace: Namespace,
+		Name:      "latency",
+		Help:      "Latency to nodes",
+	}, []string{"node", "type", "ip", "protocol"})
```

Comment:
What do you plan for the "type" to indicate? I note that it's left empty currently in both usages.

Reply:
I actually like this idea, I think it's an important differentiation.